### ISSUE TITLE
java error in generateCohortSet()

### ISSUE THREAD

**Issue by tiozab** [oxford-pharmacoepi/phenotypeR_project#26]
I have this cohort 
[preg_concepts_CPRD.json](https://github.com/oxford-pharmacoepi/phenotypeR_project/files/15118785/preg_concepts_CPRD.json)

and used the CodeToRun to use phenotypeR and I get the following error :

![image](https://github.com/oxford-pharmacoepi/phenotypeR_project/assets/94674809/eb26b5cd-2f5b-49bf-a1f4-cc55472ecb8a)

The error occurs here:

# Step 1: Get cohorts and generate them ------
# now from json, but we can do with CapR other sources 

tic(msg = "Generate Cohort Set"
cohort_set <- read_cohort_set(cohort_json_dir)
if (input$runGenerateCohort) {
  cdm <-   generateCohortSet(cdm, 
                             cohort_set,
                             name = cohorts_name,
                             computeAttrition = TRUE,
                             overwrite = TRUE)
}

the output says 2 cohorts because I left one of the pancreas cohorts in the Cohorts folder to see what it should look like. The error also occurs if I have only a cohort in there that was already in the package. 

I used the renv and installed all the packages, and installed 2 packages separately that needed installing (see error message)
![image](https://github.com/oxford-pharmacoepi/phenotypeR_project/assets/94674809/d9c58e19-d778-4296-8aa8-374f0964e456)

basically I need some help on how to execute the phenotypeR for my cohort :-) because I cannot get it to run @nmercadeb ? this is the "fake" pregnancy cohort json I generated from the concepts used in CPRD via atlas
